.github/workflows/linux.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ["stable", "nightly"]
+        version: ['stable', 'nightly']
```

Comment:
Why did you change this?

Reply:
Sorry, I think my prettier changed it automatically, will revert